src/components/AppRouter.jsx

```diff
@@ -31,6 +33,7 @@ const AppRouter = () => {
               path="group/:groupId/delete/:groupName"
               element={<GroupDeleteConfirmationModal />}
             />
+            {BarRoutes.map(BarRoute => BarRoute)}
```

Comment:
Couldn't we have used a component? I think not, because Routes necessarily expects a Route as a child, doesn't it?

Reply:
Exactly. I tried `{...BarRoutes}` but it does not work on contacts (it worked on drive 🤔). I did not take the time to understand.